### ISSUE TITLE
[NPC] ID 32973 - Дентария Серебристая Поляна <Жрица луны>

### ISSUE THREAD

**Issue by Amfinia** [VirginTeam/LegionCore#149]
**Ссылки и комментарии:**

http://ru.wowhead.com/npc=32973

**Что работает не так?**

Отсутствует NPC

**Как должно работать?**

Место положение на оффе:
![wowscrnshot_021218_221647](https://user-images.githubusercontent.com/29457913/36115025-2f807cac-104b-11e8-9859-43b14c59b5f8.jpg)

**Текущее обновление ядра:**

28.01.2018 Test Build - Trinity

**Comment by Kodekc** [comment]
NPC есть, может ты была персонажем в другой фазе?

**Comment by Amfinia** [comment]
Нет, ей надо сдать квест, который направляет игрока в эту зону (http://ru.wowhead.com/quest=28490). Возможно, ее просто нет в этой фазе. Сейчас поделаю квесты на Темном берегу, еще раз к ней загляну.

**Comment by Amfinia** [comment]
При взятии взаимозаменяемого квеста http://ru.wowhead.com/quest=26383 , дама тоже не появляется.
